src/data-workspace/data-details-sidebar/basic-information.js

```diff
@@ -27,7 +27,7 @@ const BasicInformation = ({ item, onMarkForFollowup, onUnmarkForFollowup }) => (
             <li>
                 <Tooltip content={item.lastUpdated.toString()}>
                     {i18n.t('Last updated {{- timeAgo}} by {{- name}}', {
-                        timeAgo: moment(item.lastUpdated).fromNow(),
+                        timeAgo: moment.utc(item.lastUpdated).fromNow(),
```

Comment:
I noticed that in my case, changes that I just did appeared as `an hour ago` because I am in UTC+1 while the API is returning UTC. This change makes it take into consideration the local timezone vs api timezone.